### ISSUE TITLE
 [Distance] - The i (Toggle attribution) button on the map does not show anything

### ISSUE THREAD

**Issue by kbecciv** [Expensify/App#26761]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Open request money
2. Open distance tab
3. Tap on i button on bottom right corner of the map

## Expected Result:
The toggle attribution button should open options

## Actual Result:
The toggle attribution button doesn't work when it is clicked

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android / native
- [ ] Android / Chrome
- [ ] iOS / native
- [ ] iOS / Safari
- [ ] MacOS / Chrome / Safari
- [ ] MacOS / Desktop

**Version Number:** 1.3.63.0
**Reproducible in staging?:** y
**Reproducible in production?:** n
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** Any additional supporting documentation

https://github.com/Expensify/App/assets/93399543/06df886b-b704-4b9d-8bbf-0f2bf7a94756

https://github.com/Expensify/App/assets/93399543/1b664c06-f9ea-445c-ae23-c3cfa973d36f

**Expensify/Expensify Issue URL:**
**Issue reported by:** @jo-ui
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1693894957986959

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by akinwale** [comment]
## Proposal

### Please re-state the problem that we are trying to solve in this issue.
The toggle attribution (i) button on the map does not show anything.

### What is the root cause of that problem?
The required CSS for the Mapbox 

### What changes do you think we should make in order to solve the problem?
Include the CSS for Mapbox GL: https://api.tiles.mapbox.com/mapbox-gl-js/v1.3.1/mapbox-gl.css in web.html.
On Desktop platforms, hovering will show the attribution text.
On mWeb, clicking will show the attribution text. 

### What alternative solutions did you explore? (Optional)
None.

**Comment by jo-ui** [comment]
This issue is now on the latest production too

**Comment by anmurali** [comment]
@trjExpensify - what is that `i` supposed to show?

**Comment by akinwale** [comment]
> @trjExpensify - what is that `i` supposed to show?

https://github.com/Expensify/App/assets/4160319/1e05856b-e839-466f-8e61-48e9891152cd

**Comment by PiyushChandra17** [comment]
## Proposal

### Please re-state the problem that we are trying to solve in this issue.
The toggle attribution button doesn't work when it is clicked

### What is the root cause of that problem?
In [MapView](https://github.com/Expensify/App/blob/main/src/components/MapView/responder/index.ts#L4), `onStartShouldSetPanResponder: () => true`, setting it to true (this is the root cause)

### What changes do you think we should make in order to solve the problem?
Setting this `onStartShouldSetPanResponder` to false will solve the problem

### What alternative solutions did you explore? (Optional)
NA

**Comment by trjExpensify** [comment]
>@trjExpensify - what is that i supposed to show?

Yeah, that `i` is something that comes from mapbox the map provider. I doubt it, but @neil-marcellini @hayata-suenaga is there a way for us not to show the `i` at all. I appreciate the watermark will always have to be there, like how "Google" is in the bottom-left of the map in all of my ride sharing apps that use Google maps.

**Comment by akinwale** [comment]
> Yeah, that `i` is something that comes from mapbox the map provider. I doubt it, but @neil-marcellini @hayata-suenaga is there a way for us not to show the `i` at all. I appreciate the watermark will always have to be there, like how "Google" is in the bottom-left of the map in all of my ride sharing apps that use Google maps.

Quoted from https://docs.mapbox.com/help/getting-started/attribution/:

> Maps that use Mapbox map designs, data or software usually must display the Mapbox logo and text attribution.

**Comment by trjExpensify** [comment]
Got it, so this is what they call "text attribution" is it?

**Comment by akinwale** [comment]
> Got it, so this is what they call "text attribution" is it?

Yes. The logo is displayed at the bottom-left, while text attribution is displayed at the bottom-right by default. On smaller screen widths, the bottom-right changes to an icon which you can click to reveal the text attribution.

**Comment by trjExpensify** [comment]
Cool, thanks.

**Comment by hayata-suenaga** [comment]
do you still need an input from me 😃 ?

**Comment by trjExpensify** [comment]
Ha, I think we're good thanks!

**Comment by akinwale** [comment]
@allroundexperts My [PR](https://github.com/Expensify/App/pull/27491) is ready for review. Thanks.

**Comment by Julesssss** [comment]
Ready for payment @anmurali 👍

**Comment by Julesssss** [comment]
> Ready for payment @anmurali 👍

Bumping for payment so we can close out this issue.

**Comment by anmurali** [comment]
https://github.com/Expensify/App/issues/26761#issuecomment-1727191602
It's right here. Why do I need to re-write it?

**Comment by anmurali** [comment]
Updated that entry with amounts

**Comment by JmillsExpensify** [comment]
$750 payment approved for @allroundexperts based on BZ summary.
